securebanking-openbanking-uk-rs-obie-api/src/main/java/com/forgerock/securebanking/openbanking/uk/rs/api/obie/payment/v3_1_4/domesticscheduledpayments/DomesticScheduledPaymentsApi.java

```diff
@@ -85,6 +85,9 @@
             @ApiParam(value = "Indicates the user-agent that the PSU is using.")
             @RequestHeader(value = "x-customer-user-agent", required = false) String xCustomerUserAgent,
 
+            @ApiParam(value = "Indicates if 'ReadRefundAccount' was set to 'Yes' or 'No' in the consent. Defaults to 'No'", required = false)
+            @RequestHeader(value = "x-read-refund-account", required = false) String xReadRefundAccount,
```

Comment:
@christian-brindley - if you look at the whole file then you'll see the `x-ob-account-id` that I added for scheduled payments & standing orders.

We'll need to go over the required fields across the API. I suspect we can lose some of them - like `x-jws-signature` for example.

Reply:
Good point. Lots can probably go.